docs/electron.rst

```diff
@@ -192,19 +185,18 @@ The next line is how NPM modules get bundled as part of Flatpaks:
 
 Since even simple Node.js applications depend on dozens of packages, it would
 be impractical to specify all of them as part of a manifest file. A `Python
-script <https://github.com/flatpak/flatpak-builder-tools/tree/master/npm>`__
-has therefore been developed to download Node.js packages with NPM and
+script <https://github.com/flatpak/flatpak-builder-tools/tree/master/node>`__
+has therefore been developed to download Node.js packages with NPM or Yarn and
 include them in an application's sources.
 
-The Python NPM script requires a ``package-lock.json`` file. This contains
-information about the packages that an application depends on, and can be
-generated by running ``npm install --package-lock-only`` from an application's
-root directory (the sample example contains a ``package-lock.json``, for
-reference). The script is then run as follows::
+The Python script requires a ``package-lock.json`` (or ``yarn.lock``) file. This
+file contains information about the packages that an application depends on, and
+can be generated by running ``npm install --package-lock-only`` from an
+application's root directory. The script is then run as follows::
 
-  $ python3 flatpak-npm-generator.py package-lock.json
+  $ python3 flatpak-node-generator.py npm --xdg-layout package-lock.json
```

Comment:
It probably makes sense to make `--xdg-layout` implicitly enabled, but it'll break compatibility with older apps (pre-`@electron/get`).